File: results/fine_tuning/test_output_files/question-53-pass-1.py
def isPossible(A, B):
    if (A < B):
        return 0
    for i in range(2, A + 1):
        if (A % i == 0 and B % i == 0):
            return 1
    return 0


if __name__ == "__main__":
    A = 12
    B = 15
    if (isPossible(A, B)):
        print("Yes")
    else:
        print("No")

File: feedback_pipeline/generations/cpp/files_with_feedback/question-53-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
bool isPossible(int A, int B) {
    int prime = 2;
    while (prime <= A) {
        if (A % prime == 0 && B % prime == 0) {
            return true;
        }
        prime++;
    }
    return false;
}
int main() {
    int A, B;
    cin >> A >> B;
    if (isPossible(A, B)) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
    return 0;
}